### DISCUSSION TITLE
Dashboard settings/configuration

### DISCUSSION THREAD

**Discussion by BjornLV** [SudhanPlayz/Discord-MusicBot#617] [Ideas]
Are there plans for being able to configure the server bot settings via the website dashboard? (If not maybe an good idea?) So for instance being able to change the prefix or the dj role, not with just a command in the channels but also in your discord server dashboard.

**Comment by DarrenOfficial** [comment]
For configuring prefix via dashboard is no, this is because currently we're working on v5 which only has slash command. altho on v5 there will be a way to add queue / music from the dashboard
